static/js/actions/profile.js

```diff
@@ -64,6 +64,16 @@ export const updateProfileValidation = createAction(UPDATE_PROFILE_VALIDATION,
   (username, errors) => ({ errors, username })
 );
 
+export const UPDATE_VALIDATION_VISIBILITY = 'UPDATE_VALIDATION_VISIBILITY';
```

Comment:
Can you add `profile_test.js` and add tests for these two actions? Feel free to leave out the other ones in this file for this PR